.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "external/yield-utils-v2"]
```

Comment:
yield-utils-v2 is published as an npm package as well. See [here](https://github.com/yieldprotocol/vault-v2/blob/a6b170f648ca9e25c9eb836bcf1de407e3f3bdf8/package.json#L36).

Reply:
I think it's broken, both for yarn (sort of expectedly) and npm. 
```
/tmp/y ❯❯❯ git clone https://github.com/yieldprotocol/vault-v2.git
Cloning into 'vault-v2'...
remote: Enumerating objects: 15846, done.
remote: Counting objects: 100% (3758/3758), done.
remote: Compressing objects: 100% (995/995), done.
remote: Total 15846 (delta 2816), reused 3544 (delta 2692), pack-reused 12088
Receiving objects: 100% (15846/15846), 4.19 MiB | 6.58 MiB/s, done.
Resolving deltas: 100% (11877/11877), done.
/tmp/y ❯❯❯ cd vault-v2
/tmp/y/vault-v2 [master|✔] ❯❯❯ yarn --version
1.22.11
/tmp/y/vault-v2 [master|✔] ❯❯❯ yarn install
yarn install v1.22.11
[1/4] 🔍  Resolving packages...
[2/4] 🚚  Fetching packages...
error An unexpected error occurred: "https://registry.yarnpkg.com/@yield-protocol/utils-v2/-/utils-v2-2.2.10.tgz: Request failed \"404 Not Found\"".
info If you think this is a bug, please open a bug report with the information provided in "/private/tmp/y/vault-v2/yarn-error.log".
info Visit https://yarnpkg.com/en/docs/cli/install for documentation about this command.
/tmp/y/vault-v2 [master|✔] ❯❯❯ npm install
npm WARN deprecated ethereumjs-tx@2.1.2: New package name format for new versions: @ethereumjs/tx. Please update.
npm WARN deprecated ethereumjs-common@1.5.2: New package name format for new versions: @ethereumjs/common. Please update.
npm WARN deprecated babel-eslint@10.1.0: babel-eslint is now @babel/eslint-parser. This package will no longer receive updates.
npm WARN deprecated ethereumjs-tx@1.3.7: New package name format for new versions: @ethereumjs/tx. Please update.
npm WARN deprecated ethereumjs-block@1.7.1: New package name format for new versions: @ethereumjs/block. Please update.
npm WARN deprecated eth-json-rpc-errors@2.0.2: Package renamed: https://www.npmjs.com/package/eth-rpc-errors
npm WARN deprecated ethereumjs-vm@2.6.0: New package name format for new versions: @ethereumjs/vm. Please update.
npm WARN deprecated request@2.88.2: request has been deprecated, see https://github.com/request/request/issues/3142
npm WARN deprecated debug@3.2.6: Debug versions >=3.2.0 <3.2.7 || >=4 <4.3.1 have a low-severity ReDos regression when used in a Node.js environment. It is recommended you upgrade to 3.2.7 or 4.3.1. (https://github.com/visionmedia/debug/issues/797)
npm WARN deprecated request-promise-native@1.0.9: request-promise-native has been deprecated because it extends the now deprecated request package, see https://github.com/request/request/issues/3142
npm WARN deprecated safe-event-emitter@1.0.1: Renamed to @metamask/safe-event-emitter
npm WARN deprecated eth-json-rpc-errors@1.1.1: Package renamed: https://www.npmjs.com/package/eth-rpc-errors
npm WARN deprecated ethereumjs-block@2.2.2: New package name format for new versions: @ethereumjs/block. Please update.
npm WARN deprecated har-validator@5.1.5: this library is no longer supported
npm WARN deprecated ethereumjs-account@3.0.0: Please use Util.Account class found on package ethereumjs-util@^7.0.6 https://github.com/ethereumjs/ethereumjs-util/releases/tag/v7.0.6
npm WARN deprecated ethereumjs-common@1.5.0: New package name format for new versions: @ethereumjs/common. Please update.
npm WARN deprecated ethereumjs-vm@4.2.0: New package name format for new versions: @ethereumjs/vm. Please update.
npm WARN deprecated fsevents@2.1.3: "Please update to latest v2.3 or v2.2"
npm WARN deprecated ethereumjs-blockchain@4.0.4: New package name format for new versions: @ethereumjs/blockchain. Please update.
npm WARN deprecated testrpc@0.0.1: testrpc has been renamed to ganache-cli, please use this package from now on.
npm WARN deprecated sane@4.1.0: some dependency vulnerabilities fixed, support for node < 10 dropped, and newer ECMAScript syntax/features added
npm WARN deprecated ethashjs@0.0.8: New package name format for new versions: @ethereumjs/ethash. Please update.
npm WARN deprecated mkdirp-promise@5.0.1: This package is broken and no longer maintained. 'mkdirp' itself supports promises now, please switch to that.
npm WARN deprecated multicodec@0.5.7: stable api reached
npm WARN deprecated core-js@2.6.12: core-js@<3 is no longer maintained and not recommended for usage due to the number of issues. Please, upgrade your dependencies to the actual version of core-js@3.
npm ERR! code E404
npm ERR! 404 Not Found - GET https://registry.npmjs.org/@yield-protocol%2futils-v2 - Not found
npm ERR! 404
npm ERR! 404  '@yield-protocol/utils-v2@^2.2.10' is not in the npm registry.
npm ERR! 404 You should bug the author to publish it (or use the name yourself!)
npm ERR! 404 It was specified as a dependency of 'vault-v2'
npm ERR! 404
npm ERR! 404 Note that you can also install from a
npm ERR! 404 tarball, folder, http url, or git url.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/moriarty/.npm/_logs/2021-08-12T17_33_31_827Z-debug.log
```

There's also no matches here: https://www.npmjs.com/search?q=yield-protocol%20utils

Reply:
My bad, it seems it was still private. I just published it as a public package. Version 2.2.21

https://www.npmjs.com/package/@yield-protocol/utils-v2

Reply:
done